.lgtm.yml

```diff
@@ -14,3 +14,4 @@ path_classifiers:
 queries:
 - exclude: py/not-named-self # Blocks Pydantic's @validator not accepting `self` until a better fix can be found
 - exclude: py/missing-equals # Blocks Pydantic's equivalent decorator
+- exclude: py/unsafe-cyclic-import # LGTM seems a bit pessimistic
```

Comment:
I'd like to figure this one out sometime. according to lgtm, it's my worst vice.